module/views/header_element.tpl

```diff
@@ -37,7 +37,7 @@
        <table class="table table-invisible table-condensed">
          <tbody>
             <tr>
-               %for state in "up", "unreachable", "down", "pending", "unknown", "ack", "downtime":
```

Comment:
Why did you removed pending?

Reply:
Pending is only a transient state that is not not of a great interest in the pop over ...  this state only existswhen a check has not yet been launched by Shinken. If a host/service has this state it means that your Shinken stopped just after the WebUI staterd :/

Reply:
OK.